src/lib.rs

```diff
@@ -1143,7 +1144,7 @@ externs! {
         fn __wbindgen_cb_drop(idx: u32) -> u32;
 
         fn __wbindgen_describe(v: u32) -> ();
-        fn __wbindgen_describe_closure(a: u32, b: u32, c: u32) -> u32;
+        fn __wbindgen_describe_cast() -> !;
```

Comment:
Not sure I understood why this should return `!`.

Reply:
Yeah that makes a lot of sense. Can we add some documentation around that or did I miss it somewhere else?

Reply:
Yeah I have it in the transform itself: https://github.com/wasm-bindgen/wasm-bindgen/pull/4602/files#diff-6da07b04827cd5dcb413608a7b2b1e5dca91c3d01a148cd03c110a0355cb4885R336-R339